### PR TITLE
Improve table performance of AssistedHistoryMatchingAnalysis

### LINKED ISSUE #611: AHMAnalysis plugon: Avoid freezing of plots when big data
_Remember that this is an open source project. Don't include e.g. screenshots with data that can't be publicly available._

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.
AHMAnalysis plugin freeze due to long time to load data in Dash DataTable when big amount of observations and parameters
**Describe the solution you'd like**
Improve code to reduce processing time of input data to generate the input for the Datatable and give possibility to use option argument on value filter until which KS value is to be displayed.
**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by mareco701**
This PR closes #611.

### CODE REVIEW

**Review by anders-kiaer — CHANGES_REQUESTED** [review]
Thanks @mareco701. I think we can take this in. I think we could benefit of some code restructureing in this plugin, but that is separate from this PR.

In addition to the comments below, could you add a new entry in `./CHANGELOG.md`?

File: webviz_subsurface/plugins/_assisted_history_matching_analysis.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class AssistedHistoryMatchingAnalysis(WebvizPluginABC):
     """Visualize parameter distribution change prior to posterior \
     per observation in an assisted history matching process.
     This is done by using a \
-    [KS (Kolmogorov Smirnov) test](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test) \
+    [KS (Kolmogorov Smirnov) test](https://en.wikipedia.org/wiki/KolmogorovSmirnov_test) \
```
[the remark this answers is not in the document]
Comment (mareco701) [reply webviz_subsurface/plugins/_assisted_history_matching_analysis.py]: you are right, this was a mistake from me I have put it back to what it was

File: webviz_subsurface/plugins/_assisted_history_matching_analysis.py [hunk 2]
```diff
@@ -30,22 +30,36 @@ class AssistedHistoryMatchingAnalysis(WebvizPluginABC):
     ---
 
     * **`input_dir`:** Path to the directory where the `csv` files created \
-        by the `ahm_analysis` ERT postprocess workflow are stored
+        by the `AHM_ANALYSIS` ERT postprocess workflow are stored
+    * **`ks_filter`:** optional argument to filter output to the data table based on ks value, \
+        only values above entered value will be displayed in the data table. \
+        This can be used if needed to speed-up vizualization of cases with \
+        high number of parameters  and/or observations group. Default value is 0.0.
```
[the remark this answers is not in the document]
Comment (mareco701) [reply webviz_subsurface/plugins/_assisted_history_matching_analysis.py]: this is corrected

**Comment by mareco701** [comment]
I have updated the script and added a new entry in ./CHANGELOG.md. Please have a look to see if it is ok.

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**